modules/infield_quickstart/cdf_infield_location/auth/infield_quickstart_normal_role.group.yaml

```diff
@@ -0,0 +1,69 @@
+# Role has additional write permissions to app data dataset and spaces and is the Normal InField user that can execute on checklists.
+# Typical Technician user
+name: "gp_infield_{{first_location}}_normal_users"
+sourceId: "{{infield_location_normal_users_source_id}}"
+metadata:
+  origin: "cdf-project-templates"
+  module_version: "{{module_version}}"
+capabilities:
+  # Groups - View user groups
+  - groupsAcl:
+      actions:
+        - READ
+        - LIST
+      scope:
+        all: {}
```

Comment:
That should be scoped to `currentuserscope: {}`, only root-accounts (admin to run cdf-tk for example) should get all-scope